In-house patch to remove unnecessary dependencies from Keystone's
requirements files. The specific reasons are as follows:

iso8601			Not applicable

netaddr			Not applicable

Paste			Not applicable

posix-ipc		Not applicable

pysaml2			Not applicable to Solaris

--- keystone-2015.1.2/keystone.egg-info/requires.txt.~1~	2015-10-13 10:22:09.000000000 -0700
+++ keystone-2015.1.2/keystone.egg-info/requires.txt	2016-02-02 00:30:30.677285834 -0800
@@ -2,16 +2,13 @@ pbr!=0.7,<1.0,>=0.6
 WebOb>=1.2.3
 eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
-netaddr>=0.7.12
 PasteDeploy>=1.5.0
-Paste
 Routes!=2.0,>=1.12.3
 cryptography>=0.8 # Apache-2.0
 six>=1.9.0
 SQLAlchemy<=0.9.99,>=0.9.7
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 passlib
-iso8601>=0.1.9
 python-keystoneclient<1.4.0,>=1.2.0
 keystonemiddleware<1.6.0,>=1.5.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
@@ -25,9 +22,7 @@ oslo.policy<0.4.0,>=0.3.1 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
 oauthlib>=0.6
-pysaml2
 dogpile.cache>=0.5.3
 jsonschema<3.0.0,>=2.0.0
 pycadf<0.9.0,>=0.8.0
-posix-ipc
 msgpack-python>=0.4.0
--- keystone-2015.1.2/requirements.txt.~1~	2015-10-13 10:18:02.000000000 -0700
+++ keystone-2015.1.2/requirements.txt	2016-02-02 00:30:59.005350937 -0800
@@ -6,16 +6,13 @@ pbr!=0.7,<1.0,>=0.6
 WebOb>=1.2.3
 eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
-netaddr>=0.7.12
 PasteDeploy>=1.5.0
-Paste
 Routes!=2.0,>=1.12.3
 cryptography>=0.8 # Apache-2.0
 six>=1.9.0
 SQLAlchemy<=0.9.99,>=0.9.7
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 passlib
-iso8601>=0.1.9
 python-keystoneclient<1.4.0,>=1.2.0
 keystonemiddleware<1.6.0,>=1.5.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
@@ -29,9 +26,7 @@ oslo.policy<0.4.0,>=0.3.1 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
 oauthlib>=0.6
-pysaml2
 dogpile.cache>=0.5.3
 jsonschema<3.0.0,>=2.0.0
 pycadf<0.9.0,>=0.8.0
-posix-ipc
 msgpack-python>=0.4.0
